docs/Project.toml

```diff
@@ -1,7 +1,18 @@
 [deps]
+Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
```

Comment:
Do we need all these dependencies?

Reply:
yeah because it actually runs code to make plots... I thought of making a fake results to plot with but it was extremely hacky and better to just run a 5 bus operations problem